src/tests/verify_resources.py

```diff
@@ -337,19 +339,41 @@ def test_8_res_ids(rgs, rgfns):
     # Check that resources/resource groups have a numeric ID/GroupID
 
     errors = 0
+    ridres = autodict()
+    gidrgs = autodict()
 
     for rg,rgfn in zip(rgs,rgfns):
         if not isinstance(rg.get('GroupID'), int):
             print_emsg_once('ResGrpID')
             print("Resource Group missing numeric GroupID: '%s'" % rgfn)
             errors += 1
+        else:
+            gidrgs[rg['ID']] += [rgfn]
```

Comment:
```suggestion
            gidrgs[rg['GroupID']] += [rgfn]
```
Because consistency is not our strong suit.

Reply:
Good catch!